secrets/localsecrets/localsecrets.go

```diff
@@ -78,5 +78,10 @@ func (k *keeper) Decrypt(ctx context.Context, message []byte) ([]byte, error) {
 	return decrypted, nil
 }
 
+// ErrorAs implements driver.Keeper.ErrorAs.
```

Comment:
Please add an `As` section to the package docstring just saying that it doesn't support any types for `As`.
https://github.com/google/go-cloud/blob/master/blob/memblob/memblob.go#L27-L29

Reply:
Done.